features/cli/assume_role.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Using a stored role
+
+  In order to use a role
+  As an AWS IAM user
+  I want temporary security credentials to be generated for the role
+  And the credentials to be copied to ~/.aws/credentials
+
+  Given a user has already assumed a role
+  When the user wants to assume a new role
+  Then the user must assume a credential
+  And then assume the new role
+  This workflow could be improved by assuming the user wants to use their last
+  assumed credentials to assume the new role. Alternatively, we could introduce
+  the concept of "presets" which pairs a credential with a role and then the
+  user could assume the preset instead. The preset could check if an
+  unexpired session token already exists for the credential and automatically
+  use it or issue a new token if not.
```

Comment:
Is it possible to assume one role from another assumed role? I didn't think it chained like that but I haven't checked.